internal/eval/sortedset/sorted_set.go

```diff
@@ -175,3 +175,21 @@ func (ss *Set) Get(member string) (float64, bool) {
 	score, exists := ss.memberMap[member]
 	return score, exists
 }
+
+// This func is used to remove the maximum element from the sortedset.
+// It takes count as an argument which tells the number of elements to be removed from the sortedset.
+func (ss *Set) PopMax(count int) []string {
+	result := []string{}
```

Comment:
This ensures minimum reallocs.
```suggestion
	result := make([]string,0,2*count)
```